app/controllers/AnnotationIOController.scala

```diff
@@ -64,7 +69,7 @@ class AnnotationIOController @Inject()(
     value =
       """Upload NML(s) or ZIP(s) of NML(s) to create a new explorative annotation.
 Expects:
- - As file attachment: any number of NML files or ZIP files containing NMLs, optionally with at most one volume data ZIP referenced from an NML in a ZIP
+ - As file attachment: any number of NML files or ZIP files containing NMLs, optionally with volume data ZIPs referenced from an NML in a ZIP
```

Comment:
So here also a single zip (or multiple) with multiple data-zips can be given, which is merged then, right? Could you please expand this a bit, e.g. list all the different possibilities, e.g.:
```
Each file creates an annotation. If a file is a zipfile, the following content combinations are allowed:
* 1+ nml files --> Skeleton Tracing, nmls either merged or as groups depending on createGroupForEachFile
* 1+ zip files, each containing an nml and optionally a single volume layer (is the name important here, does it need to be referenced from the nml?) --> Hybrid Tracing, nmls either merged or as groups depending on createGroupForEachFile, the volume annotations are merged (how? bucket-wise? which one wins?)
* a combination of the two above, where the nmls are merged
* 1 zip file, containing an nml and multiple volume layers --> Hybrid tracing with multiple volume annotation layers
```
That's at least the rough understanding of the intended usage I have from that route without testing it in detail^^ Is this correct? It seems that everything is actually unpacked recursively, so merging multiple layers is only prevented if they have distinct names, right?

Reply:
As discussed in person it is slightly different. This route always results in a single annotation. I added comments in code describing the different cases.